applications/ConvectionDiffusionApplication/custom_utilities/bfecc_convection.h

```diff
@@ -170,11 +176,18 @@ template<std::size_t TDim> class BFECCConvection
             }
 //             else
 //                 std::cout << "it should find it" << std::endl;
+
+            iparticle->FastGetSolutionStepValue(conv_var, 1) = iparticle->GetValue(conv_var); // Restoring the old velocity
         }
 
         KRATOS_CATCH("")
     }
 
+    void BFECCconvect(ModelPart& rModelPart, const Variable< double >& rVar, const Variable<array_1d<double,3> >& conv_var, const double substeps)
+    {
+        BFECCconvectPartially(rModelPart, rVar, conv_var, substeps, 1.0);
+    }
```

Comment:
I think it would be better to add a default argument to the current method than creating the `BFECCConvectPartially` one. If the problem is the Python export, I think we can bypass it using an auxiliary lambda function.

Reply:
Yes, I think there would be a problem with Python export. I should think about it.